exporter/templates/cat1/r3/cat1.xml

```diff
@@ -86,16 +86,9 @@
     <author>
       <time value="{{timeNow}}"/>
       <assignedAuthor>
-        <!-- id extension="Cypress" root="2.16.840.1.113883.19.5"/ -->
         <!-- NPI -->
-        {{if .CMSCompatibility}}
-          {{range .Record.ProviderPerformances}}
-            {{range .Provider.CDAIdentifiers}}
-              {{if eq .Root "2.16.840.1.113883.4.6"}}
-                <id root="2.16.840.1.113883.4.6" extension="{{.Extension}}" />
-              {{end}}
-            {{end}}
-          {{end}}
+        {{range .Record.ProviderIdentifiersForRoot "2.16.840.1.113883.4.6" "1234567893" }}
+          <id root="2.16.840.1.113883.4.6" extension="{{.}}" />
```

Comment:
Just going off of [this template in HDS](https://github.com/projectcypress/health-data-standards/blob/master/templates/cat1/r3/show.cat1.erb#L87), these provider ID's should only be there if `CMSCompatibility` is on (and same for R3.1, R4)